pandas/tests/series/test_indexing.py

```diff
@@ -1324,6 +1324,13 @@ def test_timedelta_assignment(self):
         s.loc['A'] = timedelta(1)
         tm.assert_series_equal(s, expected)
 
+        # GH 14155
+        s = Series(10*[np.timedelta64(10, 'm')])
+        s.ix[[1,2,3]] = np.timedelta64(20, 'm')
```

Comment:
Can you use `loc` here instead of `ix`? (it has the same issue, and is preferred to use over `ix`)